src/PolygonStateBridge.sol

```diff
@@ -51,6 +63,18 @@ contract PolygonStateBridge is FxBaseRootTunnel, Ownable2Step {
     constructor(address _checkpointManager, address _fxRoot, address _worldIDIdentityManager)
         FxBaseRootTunnel(_checkpointManager, _fxRoot)
     {
+        if (address(_checkpointManager) == address(0)) {
+            revert CannotSetCheckpointManagerToZero();
+        }
+
+        if (address(_fxRoot) == address(0)) {
+            revert CannotSetFxRootToZero();
+        }
+
+        if (address(_worldIDIdentityManager) == address(0)) {
+            revert CannotSetWorldIDToZero();
+        }
+
```

Comment:
Similar to the other cases, we could simplify to `error AddressZero()`, with the trade off being that the error will not specify which address is zero.